persistence-modules/pom.xml

```diff
@@ -40,8 +40,6 @@
         <module>java-jpa</module> <!-- long running -->
         <module>java-jpa-2</module>
         <module>java-jpa-3</module>
-        <!-- enable it when persistence-modules is migrated to JDK9+ -->
-        <!-- <module>java-mongodb</module>--> <!-- long running -->
```

Comment:
any reason these rows are deleted?

Reply:
Because it already included down: https://github.com/eugenp/tutorials/blob/720f6415dbd225125367df1881afb28adc86d737/persistence-modules/pom.xml#L111